packetbeat/tests/system/test_0060_flows.py

```diff
@@ -34,22 +34,22 @@ def test_mysql_flow(self):
         pprint(objs)
```

Comment:
What is this new pcap file just above (can't comment there)? Is that intentional in this PR?